### PR TITLE
Id should be a number

### PR DESCRIPTION
**Pull request by tkyi**
To reduce confusion, changing `keyName` to `key` and changing `id` type from `String` to `Number`. Also conforms with data-model (https://github.com/screwdriver-cd/data-model/blob/master/model/build.js#L7).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Save a record in the datastore
 * `config` - An `object`. Each of its properties defines your save operation
 * `config.table` - A `string`. The datastore table name
 * `config.params` - An object. Each of its properties defines the save parameters
-* `config.params.id` - A `string`. The ID to associate the data with
+* `config.params.id` - A `number`. The ID to associate the data with
 * `config.params.data` - An object. This is what will be saved in the datastore
 * `callback()` - A callback which is called when the task is completed
```
Comment (FenrirUnbound) [inline README.md]: Can you add the documentation for `callback` in the `scan()` method plz?

**Comment by nkatzman** [comment]
Are all "ids" in the data model numbers ?

**Comment by tkyi** [comment]
@nkatzman yes

File: README.md (hunk 1 above)
Comment (tkyi) [inline README.md]: @FenrirUnbound added!

**Comment by nkatzman** [comment]
As long as it's consistent 👍

**Comment by d2lam** [comment]
👍

**Pull request merged**